src/main/java/db/migration/V188__Remove_persist_lineItems_that_should_be_implicit.java

```diff
@@ -0,0 +1,83 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import java.math.BigDecimal;
+import java.sql.*;
+
+public class V188__Remove_persist_lineItems_that_should_be_implicit implements JdbcMigration {
+
+    @Override
+    public void migrate(Connection connection) throws Exception {
+        // Find all lineItems
+        PreparedStatement psLineItems = connection.prepareStatement("SELECT * FROM `LineItems`");
+
+        try {
+            connection.setAutoCommit(false);
+
+            ResultSet rsLineItems = psLineItems.executeQuery();
+
+            while (rsLineItems.next()) {
+                long lineItemId = rsLineItems.getLong("Id");
+                BigDecimal amount = rsLineItems.getBigDecimal("amount");
+                String description = rsLineItems.getString("description");
+
+                // Should not delete if description doesn't match auto-generated pattern
+                // Look for string 'buyout funds for' or 'work-life balance funds for
+                String buyoutDescription = "Buyout Funds for";
```

Comment:
In the future, you should move variables like this out of the loop so Java doesn't think it needs to create and delete them every loop.